Automattic/studio
# Improve RTL layouts

Resolves Automattic/dotcom-forge#6375

## Proposed Changes

Improves support for RTL layouts by updating compatible padding, margin, and spacing values when RTL languages are used.

## Testing Instructions

1. Change system locale to an RTL language, like Hebrew. (Steps to [change system locale on macOS](https://github.com/Automattic/dotcom-forge/issues/6375#issuecomment-2058631327), for example.)
2. Test and observe the following RTL layout-based issues are not present:


### **Change 1**: Mirror the action button icon

Before | After
-|-
<img width="138" alt="צילום מסך 2024-04-19 ב-3 06 17 אח׳" src="https://github.com/Automattic/local-environment/assets/643285/c5cf6e95-2425-475a-9d0a-ec213813a3af"> | <img width="150" alt="צילום מסך 2024-04-19 ב-3 05 16 אח׳" src="https://github.com/Automattic/local-environment/assets/643285/8439b576-0ea8-4560-be93-478e6620e65e">

### **Change 2**: Add padding to Content tab

Before | After
-|-
<img width="669" alt="צילום מסך 2024-04-19 ב-3 06 12 אח׳" src="https://github.com/Automattic/local-environment/assets/643285/e5a0545f-c652-4dc4-8ab6-e3bec4518035"> | <img width="663" alt="צילום מסך 2024-04-19 ב-3 05 26 אח׳" src="https://github.com/Automattic/local-environment/assets/643285/98d740eb-1fa4-4083-879a-e426435a6e34">

### **Change 3**: Fix label spacing and alignment on Settings tab

Before | After
-|-
<img width="491" alt="צילום מסך 2024-04-19 ב-3 12 15 אח׳" src="https://github.com/Automattic/local-environment/assets/643285/da981d10-df74-491a-a8c8-e3b7cd2ab66a"> |  <img width="651" alt="צילום מסך 2024-05-01 ב-6 16 53 אח׳" src="https://github.com/Automattic/local-environment/assets/643285/8fb8993c-bfef-46f1-9331-ebd119f1e5f5">

### **Change 4**: Updates ScreenshotDemoSite background 

Before | After
-|-
<img width="663" alt="צילום מסך 2024-04-19 ב-3 06 23 אח׳" src="https://github.com/Automattic/local-environment/assets/643285/e96e3c07-b2dd-48be-a942-b2c695b3f3de"> |  <img width="658" alt="צילום מסך 2024-04-19 ב-3 05 58 אח׳" src="https://github.com/Automattic/local-environment/assets/643285/082af078-b7ab-436e-abc6-c7b0e24e7be2">


### Known issues not addressed in this PR

* https://github.com/Automattic/dotcom-forge/issues/6662
* https://github.com/Automattic/dotcom-forge/issues/6805 (text overflow in sidebar)



## Pre-merge Checklist
